docs/guide/dependencies.md

```diff
@@ -6,11 +6,9 @@ features.
 
 | Name | License | Author |
 | :--- | :--- | :--- |
-| [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS |
+| [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS | // in src/interpreter/plugin/3rdparty
 | [Chevrotain](https://github.com/SAP/chevrotain) | Apache v2.0 | SAP SE or an SAP affiliate company |
-| [core-js](https://github.com/zloirock/core-js) | The MIT License | Denis Pushkarev |
-| [jStat](https://github.com/jstat/jstat) | The MIT License | jStat |
-| [regenerator-runtime](https://github.com/facebook/regenerator/tree/master/packages/regenerator-runtime) | The MIT License | Facebook, Inc. |
```

Comment:
Is it true that `regenerator-runtime` and `core-js` are not distributed at all in the UMD package? They are still present in `package-lock.json`.

Reply:
Yes, they are present in `package-lock.json` but with the `"dev": true` flag.

This snippet from `dist/hyperformula.js` suggests that the only dependencies bundled are:
- chevrotain
- unorm
- tiny-emitter

![image](https://user-images.githubusercontent.com/2480086/188504081-ab9f5e10-44e5-483a-8c86-a4a863b393b6.png)

Reply:
I am afraid that this snippet does not tell the whole story. If you add `pathinfo` to the Webpack output (12e22a7826dae8f0db58beefd247fc66bd42fcbe), it shows, that `hyperformula.full.min.js` and `hyperformula.min.js` contain fragments from 70 files from `./node_modules/core-js`

So, yes we should remove `regenerator-runtime` from this list, but not `core-js`.

`"dev": true` doesn't change anything, because `core-js` is copied to the bundle in the build process.

Reply:
You're right. The 3 dependencies listed above are the ones that Webpack bundles together into `hyperformula.full.js`, but there are snippets from the `core-js` lib that Babel `require`-s in the process of polyfilling.

I added `core-js` back to this table.

But as a side-note: What exactly do we call a dependency in this table?
- Libs that our users need to load before HF when using the non-full UMD build? (then it's `chevrotain`, `unorm` & `tiny-emitter`)
- Libs that help us build HF? (then it should include all the `devDependencies`)
- ...?

Reply:
The first one - what code, other than ours, does a customer run when they run HyperFormula.